server/compute/diffexp_cxg.py

```diff
@@ -3,6 +3,8 @@
 from server.compute.diffexp_generic import diffexp_ttest_from_mean_var, mean_var_n
 from server.data_cxg.cxg_util import pack_selector_from_indices
 from server.common.errors import ComputeError
+from numba import jit
```

Comment:
This is a big change in dependencies, as it will flow downstream into all users who pip install.  Has there been any research or validation that the potential for downstream breakage is acceptable?   I like the idea of using it, just noting that it is a very large change hidden behind a couple of lines of python :)

Reply:
From http://numba.pydata.org/
From there there is is a link to more info on portability.
Let me know if you think we should do something else here.

"""
Ship high performance Python applications without the headache of binary compilation and packaging. Your source code remains pure Python while Numba handles the compilation at runtime. We test Numba continuously in more than 200 different platform configurations.

Numba supports Intel and AMD x86, POWER8/9, and ARM CPUs, NVIDIA and AMD GPUs, Python 2.7 and 3.4-3.7, as well as Windows/macOS/Linux. Precompiled Numba binaries for most systems are available as conda packages and pip-installable wheels.
"""

Reply:
That seems robust. And I think we can take comfort from the fact that ScanPy has used it for quite some time.